tests/integration/cache_test.go

```diff
@@ -93,44 +197,143 @@ func testWatch(t *testing.T, kv clientv3.KV, watcher Watcher) {
 			Version:        1,
 		},
 	}
-	event5Delete := &clientv3.Event{
+	event5DeleteB := &clientv3.Event{
 		Type: clientv3.EventTypeDelete,
 		Kv: &mvccpb.KeyValue{
 			Key:         []byte("/b"),
 			ModRevision: 5,
 		},
 	}
+	event6PutFoo := &clientv3.Event{
+		Type: clientv3.EventTypePut,
+		Kv: &mvccpb.KeyValue{
+			Key:            []byte("/foo"),
```

Comment:
No need to have separate events for testing subranges, you can edit the existing `event1PutA` to make the test work with subranges.